flow/util/updateRules.py

```diff
@@ -43,7 +43,7 @@
     design = dirList[2]
     test = '{} {}'.format(platform, design)
     dataFile = os.path.join(designsDir, runFilename)
-    if os.path.exists(dataFile) and (platform != 'sky130hd_fakestack' or platform != 'src'):
+    if os.path.exists(dataFile) and (platform != 'sky130hd_fakestack' and platform != 'src' and platform not in ["intel22", "intel16", "gf12", "gf55", "tsmc65lp"]):
```

Comment:
```suggestion
    platform_list = []
    platforms_path = 'platforms'
    for item in os.listdir():
      item_path = os.path.join(platforms_path, item)
      if os.path.isdir(item_path):
        platform_list.append(item)
    if os.path.exists(dataFile) and platform in platform_list:
```

Reply:
@habibayassin 
Please test the suggestion. This way is better so we do not depend on hardcoded strings.